### DISCUSSION TITLE
angularx-social-login using version 3.5.4 in Angular 9 Application which is shown as deprecated is it supported or not?

### DISCUSSION THREAD

**Discussion by veeravetrivel** [abacritt/angularx-social-login#505] [Q&A] [answered]
angularx-social-login using version 3.5.4 in Angular 9 Application which is shown as deprecated is it supported or not?
I am getting 400 redirect_uri_mismatch and {"error":"popup_closed_by_user"}.

**Comment by Heatmanofurioso** [comment, marked as answer]
Hi @veeravetrivel

Many of the integrations with V 3.x don't work anymore since the integrations we do "with Google API, and others" don't work anymore.

We're maintaining now a version that's compatible with Angular V13.

You're more than welcome to do a PR to a V3 branch, and we could update that maintenance if you wish to.

It would basically be just cherry picking updates from our latest branch onto our latest V3.X tag, and work from there

**Reply by veeravetrivel** [reply]
ok fine I will think about it. Thank you for you reply @Heatmanofurioso :)

**Reply by ShemiNechmad** [reply]
Dear Heatmanofurioso,
I openend a new project with angular 13 and the new abacritt/angularx-social-login version,
and I'm still getting this error message from google:
Your client application uses libraries for user authentication or authorization that will soon be deprecated....

Any thoughts about that?
Thank you.
